gapic/templates/docs/index.rst.j2

```diff
@@ -2,6 +2,16 @@ API Reference
 -------------
 .. toctree::
     :maxdepth: 2
-    
+
+    {% set index_ns = namespace(has_services_api=false) %}    
+    {% for service in api.services.values() | selectattr('name') %}
+    {% if service.name == "Services" %}
+    {% set index_ns.has_services_api = True %}
```

Comment:
nit: since "API" is so overloaded, maybe `.has_service_named_services` and `.has_service_named_types`?